### DISCUSSION TITLE
It's a bit quiet in here

### DISCUSSION THREAD

**Discussion by fade2gray** [mark-wiemer/ahkpp#82] [General]
...I hope all is well with @mark-wiemer ?

**Comment by iGyver** [comment]
Maybe just a new girlfriend? :)

**Comment by mark-wiemer** [comment]
Yes, all is well. I just realized I was spreading myself too thin, so I slowed down in early March and that's turned into a bit of a full stop. I am still monitoring bug reports and feature requests, but it doesn't seem like anything major has come up.

Hopefully I'll make more progress starting mid-May, I've got lots going on until then. Thanks for your support :)

**Reply by iGyver** [reply]
I thank you very much for your support! 


It's great that you are ok. :)
